pkg/webui/components/location-form/index.js

```diff
@@ -0,0 +1,189 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { Component } from 'react'
+import { defineMessages } from 'react-intl'
+import * as Yup from 'yup'
+import bind from 'autobind-decorator'
+
+import Form from '../form'
+import Message from '../../lib/components/message'
+import Field from '../field'
+import SubmitBar from '../submit-bar'
+import Notification from '../notification'
+import ModalButton from '../button/modal-button'
+import Button from '../button'
+import toast from '../toast'
+
+import sharedMessages from '../../lib/shared-messages'
+import PropTypes from '../../lib/prop-types'
+
+import {
+  latitude as latitudeRegexp,
+  longitude as longitudeRegexp,
+  int32 as int32Regexp,
+} from '../../console/lib/regexp'
+
+const m = defineMessages({
+  deleteWarning: 'Are you sure you want to delete this location entry?',
+  deleteLocation: 'Remove location entry',
+  noLocationSet: 'There is currently no location information set',
+  updateSuccess: 'The location has been updated successfully',
+  deleteFailure: 'There was a problem removing the location',
+  deleteSuccess: 'The location has been removed successfully',
+})
+
+const validationSchema = Yup.object().shape({
+  latitude: Yup.string().required()
```

Comment:
```suggestion
  latitude: Yup.string()
    .required(sharedMessages.validateRequired)
```
otherwise the message is not translated